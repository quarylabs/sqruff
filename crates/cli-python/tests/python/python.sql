SELECT * FROM {tbl_name} 

