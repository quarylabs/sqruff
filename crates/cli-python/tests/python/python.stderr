== [tests/ui_with_python/python.sql] FAIL
L:   1 | P:  23 | LT01 | Unnecessary trailing whitespace. [layout.spacing]
L:   1 | P:  25 | LT12 | Files must end with a single trailing newline.
                       | [layout.end_of_file]
The linter processed 1 file(s).
All Finished
